src/main/java/com/revature/domain/Batch.java

```diff
@@ -26,13 +31,25 @@
 	
 	@Column(name="b_name")
 	private String name;
+	
+	@ManyToMany(cascade=CascadeType.ALL)
+	@JoinTable(name="ie_person_batch", joinColumns=@JoinColumn(name="batch_id"), inverseJoinColumns=@JoinColumn(name="person_id"))
+	private List<Person> persons;
 
 	public Batch() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)